Lib/fontbakery/specifications/general.py

```diff
@@ -116,25 +116,21 @@ def com_google_fonts_check_035(font):
 )
 def com_google_fonts_check_036(font):
   """Checking with ots-sanitize."""
+  import ots
+
   try:
-    import subprocess
-    ots_output = subprocess.check_output(
-        ["ots-sanitize", font], stderr=subprocess.STDOUT).decode()
-    if ots_output != "" and "File sanitized successfully" not in ots_output:
-      yield FAIL, f"ots-sanitize output follows:\n\n{ots_output}"
+    process = ots.sanitize(font, check=True, capture_output=True)
+  except ots.CalledProcessError as e:
+    yield FAIL, (
+      "ots-sanitize returned an error code ({}). Output follows:\n\n{}{}"
+    ).format(e.returncode, e.stderr.decode(), e.stdout.decode())
+  else:
+    if process.stderr:
+      yield WARN, (
+        "ots-sanitize passed this file, however warnings were printed:\n\n{}"
```

Comment:
the ots-sanitize tool sometimes may return 0 (success), but still print warning messages to stderr, for non-mandatory tables that are buggy and that are simply dropped when they fail to sanitize, but the font as a whole still passes.
I thought for a linter like fontbakery this is relevant information so it was worth keeping in the log.
In these cases a WARN is yielded and the ots-sanitize stderr messages are attached to the fontbakery's log.

Reply:
ehm, this if branch is currently not tested. We need a font that triggers it. Maybe one with a buggy kern table, as that's supposed to be non-mandatory